generators/server/templates/src/main/java/package/domain/_User.java

```diff
@@ -183,7 +184,7 @@ public String getLogin() {
 
     //Lowercase the login before saving it in database
     public void setLogin(String login) {
-        this.login = login.toLowerCase(Locale.ENGLISH);
+        this.login = login == null ? null : login.toLowerCase(Locale.ENGLISH);
```

Comment:
@cbornet : for your information, Sonar complains about this line [here](https://sonarcloud.io/dashboard?id=io.github.jhipster.sample%3Ajhipster-sample-application%3Asrc%2Fmain%2Fjava%2Fio%2Fgithub%2Fjhipster%2Fsample%2Fdomain%2FUser.java)
> "login" is marked "javax.validation.constraints.NotNull" but is set to null.